docs/.rebuild_patterns

```diff
@@ -2,4 +2,6 @@
 ^docs/scripts
 ^docs/static
 ^docs/*.js
-^docs/*.json
\ No newline at end of file
+^docs/*.json
```

Comment:
rebuild should be triggered when the root config files change

Reply:
This will trigger on any json file at the monorepo root right? We should probably exclude `cspell.json` in that case.

Reply:
Not sure if docs are explicitly spellchecked with that or just the code.

Reply:
yep youre correct, updating